content/general/general.md

```diff
@@ -1,9 +1,9 @@
 ---
-Title: 'General' # This is the name that appears on the hub page for this language. Pay attention to capitalization and punctuation!
-Description: 'The General section is the home to all the language-agnostic entries such as "What is Machine Learning?" and "What is a CPU?". These are the universal foundational programming terminologies that are often shared among languages and frameworks. We also added some biographies about some of the well known figures in the industry, too!'
+Title: 'General'
+Description: 'The General section is home to all programming and computer-related concepts that are language-agnostic and do not belong in one of the broader topic areas.'
```

Comment:
```suggestion
Description: 'The General section is home to all programming and computer-related concepts that are language-agnostic and do not belong in one of the focused topic areas.'
```

I think the other topics should be referred to as focused rather than broad (in relation to general)